### DISCUSSION TITLE
Issue with Docker Build: 'Changes out of order' error on Windows 11"

### DISCUSSION THREAD

**Discussion by Harshit28j** [Cloud-CV/EvalAI#4264] [Q&A]
Hi,
I'm trying to set up EvalAI on Windows 11 using Docker but getting the following error when running `docker-compose up --build`:
![image](https://github.com/Cloud-CV/EvalAI/assets/48647625/5cca61d3-dc12-4357-89fd-bff10eac3b3c)
failed to solve: changes out of order: "requirements/code_upload_worker.txt"

Could there be something with how docker handles file mounting on Windows that's causing this? Or is there a workaround for this specific error when building EvalAI?

Any help would be appreciated! I'm looking forward to start using EvalAI but been stuck on this docker build issue for a while.

**Comment by Rahulxx01** [comment]
Even I am facing the same issue

Additionally, I am getting

ERROR [django internal] load build context 

![image](https://github.com/Cloud-CV/EvalAI/assets/24786147/4192a468-aa11-496b-8592-aa124aa09ed3)

**Comment by Rahulxx01** [comment]
Issue Resolved [Django Internal]
I Updated the docker-compose version to 4.27.2 which is the latest version.
